Src/FluentAssertions/CallerIdentification/ShouldCallHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Text;
+
+namespace FluentAssertions.CallerIdentification
+{
+    public class ShouldCallHandler : IHandler
+    {
+        private const string ShouldCall = ".Should";
+        private readonly StringBuilder statement;
+
+        public ShouldCallHandler(StringBuilder statement) => this.statement = statement;
+
+        public HandlerResult Handle(char symbol)
+        {
+            if (statement.Length >= ShouldCall.Length)
+            {
+                var ldx = statement.Length - 1;
```

Comment:
🤔 What's `ldx` and `rdx`?

Reply:
I assume it's something like "left index" & "right index" - I'll rename it accordingly, hopefully that's good 🤞